### PR TITLE
Large blocks of text #8781

### PR DESCRIPTION
**Pull request by Arei1997**

### CODE REVIEW

File: conf/app.routes [hunk 1]
```diff
@@ -363,7 +363,7 @@ GET         /:srn/change-are-repayments-instalments/:index
 POST        /:srn/change-are-repayments-instalments                                                            controllers.nonsipp.loansmadeoroutstanding.AreRepaymentsInstalmentsController.onSubmit(srn: Srn, index: Max5000 = 1, mode: Mode = CheckMode)
 POST        /:srn/change-are-repayments-instalments/:index                                                     controllers.nonsipp.loansmadeoroutstanding.AreRepaymentsInstalmentsController.onSubmit(srn: Srn, index: Max5000, mode: Mode = CheckMode)
 
-GET         /:srn/interest-on-loan                                                                             controllers.nonsipp.loansmadeoroutstanding.InterestOnLoanController.onPageLoad(srn: Srn, index: Max5000 = 1, mode: Mode = NormalMode)
+GET         /:srn/                                                                             controllers.nonsipp.loansmadeoroutstanding.InterestOnLoanController.onPageLoad(srn: Srn, index: Max5000 = 1, mode: Mode = NormalMode)
```
Comment (cyeniceri) [inline conf/app.routes:366]: hi @Arei1997, Is this change done intentionally ?

Comment (Arei1997) [reply conf/app.routes:366]: Nope, let me fix that not sure why that happened

Comment (Arei1997) [reply conf/app.routes]: Should be OK now

Comment (cyeniceri) [reply conf/app.routes]: I reverted the change on /:srn/interest-on-loan endpoint in app.routes

**Review by westwater — APPROVED** [review]

**Pull request merged**